packages/api/src/gateways/eth_edge_request.ts

```diff
@@ -0,0 +1,21 @@
+import { request, Method } from "@cryptify/common/src/helpers/ApiRequest";
+import { CreateWalletRequest } from "@cryptify/common/src/requests/create_wallet_request";
+import { Injectable } from "@nestjs/common";
+import { ConfigService } from "@nestjs/config";
+
+@Injectable()
+export class EthEdgeRequest {
```

Comment:
Name it like this since its modeled after the Gateway design pattern

```suggestion
export class EthEdgeGateway {
```